### PR TITLE
Add MatekH743-Wing page

### PR DESCRIPTION
**Pull request by Hwurzburg**
will ask Sampson to review before merge

### CODE REVIEW

File: common/source/docs/common-matekh743-wing.rst [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+.. note::
+
+   Support for this board is available with ArduPilot Plane 4.0.6, Copter 4.0.4, and higher
+
+Specifications
+==============
+
+-  **Processor**
+
+   -  STM32H743VIT6  ARM (216MHz)
```
Comment (MATEKSYS) [inline common/source/docs/common-matekh743-wing.rst:22]: 480MHz

File: common/source/docs/common-matekh743-wing.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+- SERIAL7 = USER = UART6 (TX only unless :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` = 1, then RX available also)
+
+RC input is configured on the R6 (UART6_RX) pin. It supports all RC protocols, however for FPort  :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` should be set to 1 and configured as described in :ref:`FPort<common-FPort-receivers>` section.
+
+Serial port protocols (Telem, GPS, etc.) can be adjusted to personal preferences.
+
+Dshot capability
+================
+
+All motor/servo outputs are Dshot and PWM capable. However, mixing Dshot and normal PWM operation for outputs is restricted into groups, ie. enabling Dshot for an output in a group requires that ALL outputs in that group be configured and used as Dshot, rather than PWM outputs. The output groups that must be the same (PWM rate or Dshot, when configured as a normal servo/motor output) are: 1/2, 3/4/5/6,  or 7/8/9/10. Outputs 11,12, and 13 are not Dshot capable.
```
Comment (MATEKSYS) [inline common/source/docs/common-matekh743-wing.rst:83]: PWM13(LED2812) TIM1_CH1 is Dshot compable,   H7 TIM15_CH1/CH2 (PWM11/12)should work with Dshot also.

File: common/source/docs/common-matekh743-wing.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-matekh743-wing.rst]: done

File: common/source/docs/common-matekh743-wing.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-matekh743-wing.rst]: done

**Pull request merged**